### PR TITLE
Adds some more uses for ISVS

### PR DESCRIPTION
**Pull request by tkisason**
1. Adds info about how to use ISVS in design requirements
2. Adds info about how to use ISVS in procurement
3. Adds info about how to use ISVS in IoT Security Training
4. Adds info about how to use ISVS in agile security practices.

### CODE REVIEW

**Review by scriptingxss — CHANGES_REQUESTED** [review]
Really appreciate the time you've invested to help apply more color to ISVS' uses.

File: en/Using_ISVS.md [hunk 1]
```diff
@@ -37,10 +37,15 @@ Even though the standard is called a verification standard, its use goes much wi
 
 - During the design of the supporting hardware platform, the hardware platform requirements in [V5](V5-Hardware_Platform_Requirements.md) are created so that they can be used to validate that the hardware platform provides all of the functionality that is required to implement the security requirements described in the other requirement category chapters.
 
-- The requirements listed in the ISVS can be used during the requirement elicitation phase of the project.
+- The requirements listed in the ISVS can be used during the requirement elicitation phase of the project. For example, when defining security requirements in the beginning of a products secure development lifecycle, ISVS can be used as a minimum set of requirements that should guide the product's development.
```
[the remark this answers is not in the document]
Comment (tkisason) [reply en/Using_ISVS.md:40]: Accepted

File: en/Using_ISVS.md [hunk 2]
```diff
@@ -37,10 +37,15 @@ Even though the standard is called a verification standard, its use goes much wi
...
+
+- ISVS can be used in procurement of custom IoT solutions. When a third party is contracted to develop an IoT product or solution, a buyer can require that the solution is developed according to a specific ISVS security level, and request proof that the solution satisfies the required ISVS security level. 
+
+- When developing internal or external IoT security trainings, ISVS can be used to guide trainings so they contain best practices on what should be done in specific cases instead of demoing or showing common attacks against IoT systems.
```
[the remark this answers is not in the document]
Comment (tkisason) [reply en/Using_ISVS.md:44]: Accepted

File: en/Using_ISVS.md [hunk 3]
```diff
@@ -37,10 +37,15 @@ Even though the standard is called a verification standard, its use goes much wi
...
 
 - The requirements can be used to assess the overall security posture of a device's environment. It can help to define test cases, or it can be used by security professionals to assess the device's implementation.
 
+- The ISVS can be used as a framework to guide the agile development process in order to have a more secure product. Since most new IoT devices hardware will first be developed from a prototype systems or development boards, the ISVS levels of focusing on the software security first and then focusing on hardware security makes it easy to integrate it as a part of an agile security practice in an organization. The project can start by defining an end-goal ISVS level according to the projects risk assessment and then use the ISVS requirements as tickets in the development backlog. Specific features can be prioritized, and the security efforts can be easily visualized on the board. This can also be used to prioritize auditing and reviewing tasks in the organization, where a specific  requirement can be a driver for implementation, review, refactor or auditing for a specific team member and visible as "debt" in the backlog that needs to be eventually done.
```
[the remark this answers is not in the document]
Comment (tkisason) [reply en/Using_ISVS.md]: Accepted

**Comment by tkisason** [comment]
Updated according to change request. Please see if it's better now.

**Review by scriptingxss — APPROVED** [review]

**Pull request merged**